cea/interfaces/dashboard/inputs/routes.py

```diff
@@ -76,6 +76,8 @@ def route_get_building_properties():
     import cea.plots
     import cea.glossary
 
+    div = request.args.get('div', default=False)
```

Comment:
this looks like you're doing a backward-compatibility thing for the electron interface. I suggest we move all-or-nothing. and use a separate route - so it's easier to delete the old stuff when we're done with the transition.